ESPixelStick/src/service/DisplayOLED.cpp

```diff
@@ -26,38 +26,33 @@
 #include <Adafruit_GFX.h>
 #include <Adafruit_SSD1306.h>
 #include "DisplayOLED.h"
-#define SCREEN_WIDTH 128    // OLED display width, in pixels
-#define SCREEN_HEIGHT 32    // OLED display height, in pixels
-#define OLED_RESET -1       // Reset pin # (or -1 if sharing Arduino reset pin)
-#define SCREEN_ADDRESS 0x3C ///< See datasheet for Address; 0x3D for 128x64, 0x3C for 128x32
+int SCREEN_WIDTH = 128;        // OLED display width, in pixels
```

Comment:
There are two ways to make these definitions. The better way is to put them in your class definition (.h file). If you want them here, then please limit the scope by making them static const. Doing it this way pollutes the global name space. Global variables should be avoided, which means putting them in the class private scope would be the better choice.